content/docs/user/artifact_conf/index.md

```diff
@@ -18,8 +18,15 @@ To access the Artifacts section, click on **Artifacts** in the navbar.
 
 The Simulations view contains all the artifacts you have configured with the given name, team, and filename of uploaded artifact if not empty.
 
+Gatling version of the artifact is displayed in a badge next to the filename.
+
 {{< img src="artifact-table.png" alt="Artifact table" >}}
 
+{{< alert tip >}}
+Artifacts uploaded before the upload validation has been implemented may have a "corrupted" badge
+*(e.g: artifact hasn't be correctly generated)*
```

Comment:
You should add that a corrupted artifact is not usable, and they need to fix it.